src/context/config.cpp

```diff
@@ -515,6 +525,29 @@ struct config : public config_t {
         groups["services"] = component_group_t("services", services_src);
         groups["storages"] = component_group_t("storages", storages_src);
         groups["unicorns"] = component_group_t("unicorns", unicorns_src);
+        std::set<std::string> reserved_components = {
+            "services",
```

Comment:
What's wrong if I want to get "storages", "unicorns" etc by this method bypassing corresponding methods?